Charts/templates/deployment.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    app: {{ .Chart.Name }}
+    group: {{ .Chart.Name }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Chart.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Chart.Name }}
+        group: {{ .Chart.Name }}
+    spec:
+      serviceAccountName: restart-docs
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
@wcrum Let's add a health check for the container

```
        ports:
        - containerPort: 80
        livenessProbe:
          httpGet:
            path: /  
            port: 8080
          initialDelaySeconds: 15
          periodSeconds: 20
      ```
      
      Something like this, but verify my config as I pulled it from memory 🙈